File: edukate-common/src/main/java/io/github/sanyavertolet/edukate/common/utils/AuthHeaders.java
package io.github.sanyavertolet.edukate.common.utils;

import lombok.RequiredArgsConstructor;

@RequiredArgsConstructor
public enum AuthHeaders {
    AUTHORIZATION_NAME("X-Authorization-Name"),
    AUTHORIZATION_ROLES("X-Authorization-Roles"),
    AUTHORIZATION_STATUS("X-Authorization-Status"),
    AUTHORIZATION_ID("X-Authorization-Id"),
    ;

    public final String headerName;
}

File: edukate-common/src/main/kotlin/io/github/sanyavertolet/edukate/common/utils/AuthHeaders.kt
package io.github.sanyavertolet.edukate.common.utils

enum class AuthHeaders(val headerName: String) {
    AUTHORIZATION_NAME("X-Authorization-Name"),
    AUTHORIZATION_ROLES("X-Authorization-Roles"),
    AUTHORIZATION_STATUS("X-Authorization-Status"),
    AUTHORIZATION_ID("X-Authorization-Id"),
    ;
}
